core/src/main/java/com/rim/IceField/ItemBase.java

```diff
@@ -18,6 +18,8 @@ public ItemBase(String tag) {
         this.active = false;  //the item that does not belog to the inventory, cannnot be used
     }
 
+    public abstract boolean useItem(PlayerBase player);
+
```

Comment:
Why is player in the parameter? The use item can be called by the player, no need to have the player in the parameter

Reply:
Because I need it in the method implementation in the derived classes